README.md

```diff
@@ -33,6 +33,18 @@ Detailed instructions for installing and running the index meta-database and the
 The latest stable version of this package can be obtained from [PyPI](https://pypi.org/project/optimade) `pip install optimade`.
 The latest development version of this package can be installed from the master branch of this repository `git clone https://github.com/Materials-Consortia/optimade-python-tools`.
 
+## Supported OPTIMADE versions
+
+Each release of optimade-python-tools only targets one version of the OPTIMADE specification, summarised in the table below.
```

Comment:
``````suggestion
Each release of the `optimade` package in the OPTIMADE Python tools repository only targets one version of the OPTIMADE specification, summarised in the table below.
```